### PR TITLE
Add format button to routine, event and trigger.

### LINKED ISSUE #18558: Missing sql query `Format` button on Routines, Events and Triggers
### Describe the bug

`Format` button is missing on Routines, Events and Triggers modals. I don't know if this is intended or omitted. Since linting is executed, a format button should be present.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/database/routines&db=sakila&server=1
2. Click on `Create new routine`
3. See missing `Format` button

1. https://demo.phpmyadmin.net/QA_5_2/index.php?route=/database/events&db=sakila&server=1
2. Click on `Create new event`
3. See missing `Format` button

1. https://demo.phpmyadmin.net/QA_5_2/index.php?route=/database/triggers&db=sakila&server=1
2. Click on `Create new trigger`
3. See missing `Format` button

### Expected behavior

A `Format` button for the query should be added.

### Screenshots

https://github.com/phpmyadmin/phpmyadmin/assets/25424343/2ae36c36-d5d9-4ec5-a679-7ea00ee600ff

### Server configuration

- phpMyAdmin version: 5.2.2-dev, 6.0.0-dev

### PR DESCRIPTION
**Pull request by NOPR9D**
### Description

Add format button to Create/Edit modal for Routine, Event and Trigger.

Fixes https://github.com/phpmyadmin/phpmyadmin/issues/18558

### CODE REVIEW

**Comment by NOPR9D** [comment]
is the DCO check drunk ?

here i got this message : 
"Expected "NOPR9D [boucham-amine@mylabz.xyz](mailto:boucham-amine@mylabz.xyz)", but got "Boucham Amine [boucham-amine@mylabz.xyz](mailto:boucham-amine@mylabz.xyz)".

and on the fork he say : 
"This commit has an invalid Signed-Off-By line, please use your full name for legal compliance reasons.
Please check [Developer's Certificate of Origin](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO)
Here is a valid example: Signed-off-by: Jane Smith <jane.smith@example.org>"
https://github.com/NOPR9D/phpmyadmin/commit/2e4ed64ce715bb43920eb3cc8cb4ed4315c037e0#commitcomment-137057332

**Comment by williamdes** [comment]
> is the DCO check drunk ?
> 
> here i got this message : "Expected "NOPR9D [boucham-amine@mylabz.xyz](mailto:boucham-amine@mylabz.xyz)", but got "Boucham Amine [boucham-amine@mylabz.xyz](mailto:boucham-amine@mylabz.xyz)".
> 
> and on the fork he say : "This commit has an invalid Signed-Off-By line, please use your full name for legal compliance reasons. Please check [Developer's Certificate of Origin](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO) Here is a valid example: Signed-off-by: Jane Smith [jane.smith@example.org](mailto:jane.smith@example.org)" [NOPR9D@2e4ed64#commitcomment-137057332](https://github.com/NOPR9D/phpmyadmin/commit/2e4ed64ce715bb43920eb3cc8cb4ed4315c037e0#commitcomment-137057332)

Nope, see https://patch-diff.githubusercontent.com/raw/phpmyadmin/phpmyadmin/pull/18916.patch
Your git settings are maybe wrong (depending on what you expect)

That said, you posted the DCO line in the commit. For me this is all good. Nevermind 😄

File: resources/templates/database/events/editor_form.twig [hunk 1]
```diff
@@ -88,7 +88,8 @@
           <td>
             <textarea name="item_definition" rows="15" cols="40">
               {{- event.item_definition -}}
-            </textarea>
+            </textarea><br>
+            <input type="button" value="Format" id="format" class="btn btn-secondary button sqlbutton">
```
Comment (williamdes) [inline resources/templates/database/events/editor_form.twig:92]: Please use the trans filter like other strings of the file

```suggestion
            <input type="button" value="{{ 'Format'|trans }}" id="format" class="btn btn-secondary button sqlbutton">
```

**Comment by NOPR9D** [comment]
> > is the DCO check drunk ?
> > here i got this message : "Expected "NOPR9D [boucham-amine@mylabz.xyz](mailto:boucham-amine@mylabz.xyz)", but got "Boucham Amine [boucham-amine@mylabz.xyz](mailto:boucham-amine@mylabz.xyz)".
> > and on the fork he say : "This commit has an invalid Signed-Off-By line, please use your full name for legal compliance reasons. Please check [Developer's Certificate of Origin](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO) Here is a valid example: Signed-off-by: Jane Smith [jane.smith@example.org](mailto:jane.smith@example.org)" [NOPR9D@2e4ed64#commitcomment-137057332](https://github.com/NOPR9D/phpmyadmin/commit/2e4ed64ce715bb43920eb3cc8cb4ed4315c037e0#commitcomment-137057332)
> 
> Nope, see https://patch-diff.githubusercontent.com/raw/phpmyadmin/phpmyadmin/pull/18916.patch Your git settings are maybe wrong (depending on what you expect)
> 
> That said, you posted the DCO line in the commit. For me this is all good. Nevermind 😄

Thanks for your explanation, i will try to fix that for next time !

**Review by MauricioFauth — APPROVED** [review]

**Comment by MauricioFauth** [comment]
Merged. Thank you for your contribution.

**Comment by NOPR9D** [comment]
Hurray, thanks!

Had a good day!

**Pull request merged**